src/VirtualClient/TestResources/expectedoltp-c.fio2.jobfile

```diff
@@ -18,6 +18,7 @@ time_based=1
 [randomreader]
 new_group
 bs=8K
```

Comment:
Looks like you changed both fio1 and fio2 to assume 2 disks. Could you keep the original 1 disk test and then add a 2 or even 8 disks test?